en/docs/guides/login/use-certificates-with-applications.md

```diff
@@ -0,0 +1,20 @@
+# Use Certificates with Applications
+
+If you want to configure your application to use a public certificate instead of the default JWKS endpoint, follow the steps given below.
```

Comment:
Let's mention the use cases where we need this feature.

1. OIDC Token Encryption
2. Request Object flow